tests/EpisodeTest.php

```diff
@@ -4,36 +4,311 @@
-            'status'                => 0
+        factory('Suyabay\Episode', 5)->create();
+
+        $repository = new EpisodeRepository();
+        $getAllEpisodes = $repository->getAllEpisodes()->toArray();
+        $this->assertTrue(is_array($getAllEpisodes));
+        $this->assertArrayHasKey('episode_name', $getAllEpisodes[0]);
+        $this->assertArrayHasKey('episode_description', $getAllEpisodes[0]);
+    }
+
+    /**
+     * Assert that EpisodeRepository's findEpisodeById returns an
+     * episode by the passed ID.
+     *
+     * @return void
+     */
+    public function testFindEpisodeById()
+    {
+        $episode = factory('Suyabay\Episode')->create();
+
+        $repository = new EpisodeRepository();
+        $getAllEpisodes = $repository->findEpisodeById($episode['id'])->toArray();
+
+        $this->assertTrue(is_array($getAllEpisodes));
+        $this->assertArrayHasKey('episode_name', $getAllEpisodes);
+        $this->assertArrayHasKey('episode_description', $getAllEpisodes);
+    }
+
+    /**
+     * Assert that EpisodeRepository's findEpisodeWhere returns an
+     * episode by the passed value.
+     *
+     * @return void
+     */
+    public function testFindEpisodeWhere()
+    {
+        $episode = factory('Suyabay\Episode')->create();
+
+        $repository = new EpisodeRepository();
+        $getEpisode = $repository->findEpisodeWhere('episode_name', $episode['episode_name'])->get()->toArray();
+
+        $this->assertTrue(is_array($getEpisode[0]));
+        $this->assertArrayHasKey('episode_name', $getEpisode[0]);
+        $this->assertArrayHasKey('episode_description', $getEpisode[0]);
+    }
+
+    /**
+     * Assert that EpisodeRepository's getEpisodes returns all epsiodes whose
+     * id matches one provided in the passed array.
+     *
+     * @return void
+     */
+    public function testGetEpisodes()
+    {
+        factory('Suyabay\Episode', 5)->create();
+        $repository = new EpisodeRepository();
+        $episodes = $repository->getEpisodes([1,2,3,4])->toArray();
+
+        $this->assertTrue(is_array($episodes));
+        $this->assertArrayHasKey('episode_name', $episodes['data'][0]);
+        $this->assertArrayHasKey('episode_description', $episodes['data'][0]);
+    }
+
+    /**
+     * Assert that EpisodeRepository's  createEpisode adds a new
+     * episode into the database.
+     *
+     * @return void
+     */
+    public function testCreateEpisode()
+    {
+        $repository = new EpisodeRepository();
+        $episode = $repository->createEpisode([
+            'episode_name' => 'Swanky new Episode',
+            'episode_description' => 'Swanky New episode description',
+            'view_count'            => 10,
+            'image'                 => "http://goo.gl/8sorZR",
+            'audio_mp3'             => "http://goo.gl/LkNP5M",
+            'channel_id'            => 1,
+            'status'                => 0,
+            'likes'                 => 10
```

Comment:
**Code Quality - Critical**

This block of code - from the creation of an instance of Episode Repository to the calling of `createEpisode` method is repeated also in `testUpdateEpisode` method. 

Don't you think it should be abstracted into a function of its own to avoid duplication?